react/lesson1.md

```diff
@@ -36,175 +91,431 @@ cd my-app/
 npm start
 ```
 
+# Our first component
 
-# JSX
-- Uses [JSX](https://facebook.github.io/react/docs/introducing-jsx.html)
+Components in React can be defined by just declaring a function that return some JSX.
```

Comment:
A short explanation of what JSX would be good here e.g. JSX is HTML-like code that you can include in JavaScript files.

Reply:
I might rename JSX here to HTML. JSX is explained a bit later?